core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -821,7 +823,7 @@ func (eb *EthBroadcaster) tryAgainWithNewLegacyGas(ctx context.Context, lgr logg
 		return errors.Wrap(err, "tryAgainWithNewLegacyGas failed"), true
 	}
 
-	if err = saveReplacementInProgressAttempt(eb.q, attempt, &replacementAttempt); err != nil {
+	if err = eb.orm.SaveReplacementInProgressAttempt(attempt, &replacementAttempt); err != nil {
```

Comment:
Why does this one not use `pg.WithQueryer` like above?

Reply:
`getInProgressEthTx` is only used once in `handleAnyInProgressEthTx` and it gets the queryer from `eb.q` (so the underlying Queryer is a `pg.Q`) which is equivalent just using the orm's `pg.Q` instead.

I should probably change line 449 to not use pg.WithQueryer` then 👍

Reply:
To be more concise: if a method directly uses `eb.q` which is of type `pg.Q` then it is can be replaced with using the orm.q of type `pg.Q` because they both use equivalent queryers. We'd only need to pass in queryers if they are different (like if you want a new context or underlying Queryer)